user-doc/annotation.md

```diff
@@ -401,12 +402,53 @@ the alternative one has modified the representation of each entity in some way.
+- `{`... `"identifier": ` _column_ ...`}`: The _column_ stores a resolvable, globally unique, identifier for the asset. In its simplest form this MAY be satisfied by supplying a URL to the resource. The identifier MUST be _globally unique_ which is satisfiable by any fully qualified HTTP URL, for example. In any case the identifier MUST be _resolvable_ meaning that a client can retrieve the identified asset either directly or indirectly through a resolution service. More formal identifier schemes may also be considered such as MINID, DOI, ARK, or PURL. It is up to client tooling to recognize and resolve identifiers in such schemes.
+- `{`... `"accession_number": ` _column_ ...`}`: The _column_ stores the "accession number" for the asset. The scheme for an accession number is defined by an application (i.e., a science data repository, data hub, community data resource service, etc.). The accession number MUST be guaranteed to be unique _within_ the application. They MAY be globally unique but generally are not, in practice. As an example, accession numbers often take the following form `PREFIX + SERIAL NUMBER [ '.' + REVISION NUMBER ]`, such as `GDS6248`. The example given here is non-normative.
+- `{`... `"name": ` _pattern_ ...`}`: A name for the asset to be derived by a [Pattern Expansion](#pattern-expansion) on _pattern_. The name may be used to represent a file name, but is not required to do so.
+- `{`... `"bytes": ` _column_ ...`}`: The _column_ stores the file size in bytes of the asset. It should be an integer typed column.
+- `{`... `"sha256": ` _column_ ...`}`: The _column_ stores the checksum generated by the 'sha256' cryptographic hash function.
+- `{`... `"creator": ` _column_ ...`}`: The _column_ stores the _creator_ of the asset. The _column_ MAY be defined as a foreign key that references a person or user table in the model. The usual model introspection applies.
+- `{`... `"description": ` _pattern_ ...`}`: A description for the asset to be derived by a [Pattern Expansion](#pattern-expansion) on _pattern_.
+- `{`... `"date": ` _column_ ...`}`: The _column_ stores the date that the asset metadata was recorded in the system.
+- `{`... `"mediatype": ` _column_ ...`}`: The _column_ stores the Media Type of the asset, for example `application/dicom`. The format of its contents MUST conform to the [Media Type RFC](https://tools.ietf.org/html/rfc6838). The values used SHOULD come from the [list of Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml) registered with IANA.
+- `{`... `"format": ` _column_ ...`}`: The _column_ stores the identifier for the file format that the asset _conforms to_, for example `http://edamontology.org/format_3548`. See the [EDAM Ontology](http://edamontology.org) for a listing of file formats used in bioinformatics.
```

Comment:
we could say 'globally unique identifier'. i don't know whether it should be specified as a URN or not.